docs/zh_CN/dgs/dataloader.md

```diff
@@ -1,3 +1,183 @@
-# 从数据源加载数据
+# Loading From Your Data Source
 
-（comming soon...）
+By using kafka queues as the entry for streaming graph updates, the Dynamic-Graph-Service (abbreviated as dgs) can
+decouple the different data sources and sampling workers.
+Users can develop their own data loaders to ingest the source data and push graph updates into the output kafka queues
+for further consuming.
+
+## Rules to Follow
+
+When processing source data in the dataloader, these rules should be followed:
+- The graph schema from source data must be consistent with dgs.
+- A produced kafka message must be a batch of graph updates, defined as a flatbuffers table [RecordBatchRep](https://github.com/alibaba/graph-learn/blob/master/dynamic_graph_service/fbs/record.fbs).
+- Graph updates are partitioned for sampling workers, all records in one batch must have the same data partition id.
```

Comment:
for sampling workers --> among sampling workers

Reply:
done